### PR TITLE
Album buy feature

### PR DESCRIPTION
**Pull request by HarryPerez**
## Trello Card
https://trello.com/c/Mj210hD5/31-comprar-album

### CODE REVIEW

File: app/controllers/albums.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        throw errors.userNotFound(`The user with email: ${userData.email} could not be found`);
+      } else {
+        return albumsService.findBy({ albumId: req.params.id, userId: userFound.id }).then(albumFound => {
+          if (albumFound) {
+            logger.info(`The album with id: ${req.params.id} was already bought by the user`);
+            throw errors.albumAlreadyBought(
+              `The album with id: ${req.params.id} was already bought by the user`
+            );
+          } else {
+            return albumsService.findByIdFromApi(req.params.id).then(albumInfo => {
```
Comment (molinajulian) [inline app/controllers/albums.js:23]: Maybe `getBy` ?

File: test/album.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  .get('/albums?id=2')
+  .reply(200, [
+    {
+      userId: '1',
+      id: '2',
+      title: 'Batman rules'
+    }
+  ]);
+
+describe('albums controller', () => {
```
Comment (molinajulian) [inline test/album.js:39]: Delete controller

**Review by molinajulian — APPROVED** [review]
Resolve conflicts please

File: app/controllers/albums.js (hunk 1 above)
Comment (HarryPerez) [reply app/controllers/albums.js:23]: I will do it in the next pr 👍

File: test/album.js (hunk 2 above)
Comment (HarryPerez) [reply test/album.js:39]: I will do it in the next pr 👍

**Pull request merged**